src/pages/GradePage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button, Text, theme } from '@team-entry/design_system';
+import styled from '@emotion/styled';
+import BeforeGraduate from '../assets/BeforeGraduate.svg';
+import Graduate from '../assets/Graduate.svg';
+import BlackExam from '../assets/BlackExam.svg';
+import { Link } from 'react-router-dom';
+
+const GradePage = () => {
+  const list = [
+    {
+      img: BeforeGraduate,
+      title: '졸업예정자',
+      text: '아직 중학교를 졸업하지 않은\n졸업자를 칭합니다',
+      type: 'beforegraduate',
+    },
+    { img: Graduate, title: '졸업자', text: '이미 중학교를 졸업한 \n지원자를 칭합니다.', type: 'graduate' },
+    { img: BlackExam, title: '검정고시', text: '(중학교 졸업학력)\n\n', type: 'blackexam' },
+  ];
+
+  return (
+    <_Container>
+      <_Wrapper>
+        <Text color="black900" size="header1">
+          성적 산출 유형 선택
+        </Text>
+        <_Line></_Line>
```

Comment:
```suggestion
        <_Line />
```
로 self closing 하는건 어떤가요